Dockerfile-csi-controller

```diff
@@ -12,23 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM centos:7
-RUN yum --enablerepo=extras -y install epel-release && yum -y install python36-pip
+FROM registry.access.redhat.com/ubi8/python-36:1
+MAINTAINER IBM Storage
+
+###Required Labels
+LABEL name="IBM Block CSI Driver Controller" \
```

Comment:
IBM block storage CSI driver

I don't know if controller should be here or not